pkgs/tools/virtualization/extra-container/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib, nixos-container, openssh, glibcLocales, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "extra-container";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "erikarvstedt";
+    repo = pname;
+    rev = version;
+    sha256 = "12xqa11v583ajdv51g1833rxvrndmly9h4r62wc3llm8xs6k7ais";
+  };
+
+  buildCommand = ''
+    install -D $src/extra-container $out/bin/extra-container
+    patchShebangs $out/bin
```

Comment:
```suggestion
    patchShebangs $out/bin
    install -D $src/extra-container $out/bin/extra-container
```

Reply:
We have to copy the script file to `bin` before patching it.

Reply:
Can you explain why?

Reply:
`patchShebangs $out/bin` (line 1) would fail because `$out/bin` doesn't exist.

Reply:
I mena ``patchShebangs $src/extra-container``.

Reply:
In this case it will fail because `$src` is a read-only store path.